docker/README.md

```diff
@@ -284,8 +288,8 @@ ROARR_LOG: false
 
 FiftyOne Teams version 1.2 and higher supports using text
 similarity searches for images that are indexed with a model that
-[supports text queries](https://docs.voxel51.com/user_guide/brain.html#brain-similarity-text).
-To use this feature, use a container image containing
+[supports text queries](https://docs.voxel51.com/user_guide/brain.html#brain-similarity-text)
+. To use this feature, use a container image containing
```

Comment:
it's ok to have punctuation on the line of a link
```suggestion
[supports text queries](https://docs.voxel51.com/user_guide/brain.html#brain-similarity-text).
To use this feature, use a container image containing
```

Reply:
Well now, see... this 'standard' seems very flexible and very difficult to pattern check for.